crates/core/tedge/src/cli/connect/command.rs

```diff
@@ -394,9 +394,12 @@ fn new_bridge(
 
     if bridge_config.cloud_name.eq("c8y") {
         println!("Creating the device in Cumulocity cloud.\n");
+        let tedge_config = tedge_config()?;
```

Comment:
The commands have been designed to avoid this: a command should not retrieve the config, the latter being provided when the command is built.

* See how a `ConnectCommand` is build from a `TEdgeConnectOpt`.
* The `ConnectCommand` holds a `TEdgeConfigRepository` that has just to be loaded.
* You can revisit the `ConnectCommand` to hold directly the `TEdgeConfig` rather than the `TEdgeConfigRepository`.